### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#9] [General]
# about/

My Blog

https://guozheng.rbind.io/about/

**Comment by yihui** [comment]
不用谢！希望你可以坚持写上十年！：）

**Reply by residualsun1** [reply]
好嘞！虽然我的初衷是给未来的自己留下回忆，但大家的每一次留言也是我坚持下去的重要鼓励。

**Comment by Liam-Zhong** [comment]
好漂亮的博客

**Reply by residualsun1** [reply]
感谢你的肯定😄！

**Comment by XiangyunHuang** [comment]
我看了好些文章，发现博主好有个性，好真诚，希望一直坚持下去，终有一天，你会出现在书店最耀眼的位置。

**Reply by XiangyunHuang** [reply]
对了，有R语言编程的问题，欢迎来统计之都论坛，里面有很多我佩服的技术牛人。我也想了解社会学，你能推荐一本社会学的书给我吗？我是学统计学的，对社会学毫无基础的，希望是一本比较适合外行人，对你影响比较大的一本书。两本也行，多了我怕我一时读不了。上次，让人给推荐一本心理学的书，还是两年前，最近，我才把读后感给写完。

**Reply by residualsun1** [reply]
湘云你好！非常高兴能收到你的留言！

说来比较惭愧，我虽然念社会学，但直到大二第二学期加入了老师的读书会才开始有意识地阅读，而由于当时我们读的都是人类学的民族志，加上我也喜欢这方面的内容，因此就一直沿着这个方向继续学习了。所以，对于社会学，严格来说我也还是一个十足的小学徒，自觉对它的认识还处于很浅显很浅显的阶段。不过，我在统计之都论坛上发现一位念人口学的博主留下了[网址](https://blog.yanyuteng.cn/)，他本硕都在念社会学，期间的阅读量让我很佩服，还做了许多读书笔记或感想，我想或许你也可以通过他的视角感受一下社会学。

但是，得到你的邀请推荐我也觉得很荣幸哈哈。那我必须推荐几本，包括美国医学人类学家凯博文写的《[道德的重量：在无常和危机前](https://book.douban.com/subject/2567964/)》和法国社会学家迪迪埃·埃里蓬写的《[回归故里](https://book.douban.com/subject/34942789/)》。两本书，我其实只读过前面一本，对后一本有所了解，也准备读。前一本讲的是凯博文眼中7个普通人的故事，关于道德困境、经验冲突、所谓善恶、苦难，以诚恳的态度，缓缓地叙述，直到现在也让我念念不忘，还很好读（优先推荐）；后一本是迪迪埃从工人出身的家庭走向知识分子后，用社会学的视角回顾自己的过往，关于社会阶级、家庭出身、身份认同等等，寻找自我，我想这种带有个人生命历程的回顾和思考或许会比较容易引起读者共鸣，因而会比较好读，同时还带有社会学的视角，就也分享于你。不过，我还想推荐一本我读过的，自觉也比较好读的，带有明显社会学分析视角的研究著作：阎云翔的《[中国社会的个体化](https://book.douban.com/subject/10048862/)》，用社会学的视角看待一个东北农村的社会变迁，材料大多是日常的，读起来很有生活感，对于理解今天的一些状况也具有启发性，或许你也会喜欢。

好像三本有点多，不过我想范围大一些，可以有个对比参照，第一本偏向故事性、叙述；第二本偏向个人人生的回顾、反思；第三本则是学术研究色彩较浓；你按照自己的情况来选择阅读就好啦。没能准确推荐某一本，一方面是因为我也还是个小学徒，另一方面则是觉得社会学或人类学，它们对我的影响都是阅读的积累，一种年深月久的影响。因此我暂时还找不到某本对我影响特别大的，但就阶段性而言，《道德的重量》中的思考和叙述让我很难忘。

最后，很感谢你的鼓励！我一直把你、袁凡和益辉当作统计之都的统计方向的榜样，统计之都也是我常逛、关注的地方。虽然我现在没有继续学习R了，但每每看到袁凡发布R的学习笔记，我心里也会再次被吸引。有了blogdown后，我想我不仅会坚持写作，也会试着在写作上拓展范围，R的学习就是我心里的一个方向。另，我也时不时拜访你的博客，最近印象深刻的是看到你谈人生太短，希望能通过行动为自己过好后2/3，我觉得我的1/3也很快要结束了，意识到这件事情后对你写的文字深以为然，共勉 : D。

**Reply by XiangyunHuang** [reply]
谢谢你的荐书，考虑很全面。

**Reply by XiangyunHuang** [reply]
我逛了几个大书店，想着线下看看再买，可能是这几本书出版的年份较早，一本都没有。不过，网上电子版还是可以弄到的。

**Reply by XiangyunHuang** [reply]
我平时，逛书店比较多，一般只看纸质版，除非书在国内不卖，一个小癖好。所以，我有点好奇，你看了那么多书大都是电子版吗？

**Reply by residualsun1** [reply]
我也喜欢看纸质书，《道德的重量》和《中国社会的个体化》应该是因为出版的时间比较久了，不容易在实体书店见到，我是在网上买的影印版。不过我平时也会经常收藏想看的书籍的电子版，图方便的时候就用电子设备看，但还是钟爱纸质。

不知道你去的是哪些书店？我之前去了南京的先锋书店，回广州后又去了扶光书店、1200book&bed等[独立书店](https://zh.wikipedia.org/zh-cn/%E7%8D%A8%E7%AB%8B%E6%9B%B8%E5%BA%97)，发现这些书店都很有人文气息，人文、社科类的书籍种类特别丰富，内部的装潢也是个性化、诗意的，我特别喜欢，就是书籍价格太贵了……所以我大多数时候有想看的书都到网上买，没有原版书的话就看影印版。

《回归故里》2020年出版，还是挺新的，按照我的经验，在独立书店找到的概率要比传统书店更大。

**Reply by XiangyunHuang** [reply]
之前，我去了三联韬奋书店（中国美术馆店、三里屯店）、Pageone ，我昨天已经网上买了《回归故里》这本。我跟你一样，绝大部分线下看，网上买，偶有觉得环境真不错，就顺手买了。这几家店都还不小。逛书店逛出兴趣之后，想把北京大部分有名的书店都去一下，现在打卡不下 10 家了。

北京这边 Pageone（前门店）和三联韬奋（美术馆店）对读者蛮友好的，里面有不消费也可坐看的地方，而且视野空旷。那些必须消费才有座的书店，商业化太浓，相比而言，书也少一点。

**Reply by XiangyunHuang** [reply]
对了，钟书阁（西单老佛爷店）也不错。顺便吐槽一下，网红书店模范书局·诗空间（教堂改造的）我感觉就不太好，大堂都是收费的咖啡座，四周书很少，而且，可以取阅的比较旧，不可以取阅的在玻璃柜里。周围的环境布置只适合拍照，为书店打广告。

**Reply by residualsun1** [reply]
> 三联韬奋书店（中国美术馆店、三里屯店）、Pageone 、钟书阁（西单老佛爷店）

真好，如果以后有机会去北京的话，我也要去这些书店逛一逛（收到安利）。说到不消费也可坐看，我记得扶光和1200也是。

> 我昨天已经网上买了《回归故里》这本

行动派哈哈，希望阅读愉快！

**Reply by XiangyunHuang** [reply]
独立书店，又看到一新词，现在的书店若不在装饰上下功夫，真吸引不到人。王府井和中关村那种书店大超市，在我11年刚来北京读大学的时候，是多么新奇的体验。除了图书馆以外，我感觉现在 PageOne、 三联和钟书阁，是北京书界三巨头。

**Reply by residualsun1** [reply]
独立书店应该就是相对于王府井和中关村这类大型书店而言，相对来说，独立书店应该会更注重读者的体验，我也是去了扶光和1200之后才有了这个概念。不过我没去过北京，如果去了，估计也会觉得王府井和中关村的书店体验很新奇。

搜了一下，钟书阁被美国wired杂志评为世界最美书店前十名，在佛山也有分店，之后找个时间从广州坐地铁去看看，顺便在佛山玩玩哈哈。另，我在武汉去过一家三联书店（江汉路），因为《三联生活周刊》，还有三联的出版社出版的书籍，我比较喜欢三联。

**Reply by XiangyunHuang** [reply]
你已经见识过南京和广州的大书店，恐怕不会觉得王府井和中关村图书大厦有什么新奇了。我那个新奇是相对于我读大学以前看过的县城书店。如果还能感觉到新奇一定是陌生感和不同的设计，特别是设计，而后者往往要去你说的独立书店或图书馆（又特别是新建的）。

**Reply by XiangyunHuang** [reply]
《回归故里》看完了，如果我之前没有读《蛤蟆先生去看心理医生》和写完读后感，我很可能也会失眠。这本书同样在很多地方回顾自己的童年生活，这简直是在点我，并且对童年经历进行再思考，到了成年后，他因同性恋遭遇种种苦难，他把经历写下来，又分析，又上升到社会学、阶级等高度。尽管社会学方面和同性恋方面，我都没啥共鸣，特别是诸多对社会学著作的理论的引用，让我感觉枯燥，因为我找不到他与这些理论的联系。总的来说，这是本值得一读的书。

作者在最终章中说，得亏自己在写作过程中读了很多关于回归主题的书籍，中和了过重的感情负担，不然，直面真实，他将不堪重负。我当然没有他那么大的心理负担，但这份属于我的负担对我来说，也是不小。

**Reply by residualsun1** [reply]
没想到《蛤蟆先生去看心理医生》对你影响这么大，我到现在也还没遇到过能对自己产生极大影响的书籍。

另，我想，阅读社会学和人类学这一类的书籍可能需要一个适应的过程。我大一的时候许多书都看不进去（还记得大一开学第一晚，我就抱着吉登斯的教材《社会学》到图书馆看……但……看不进去），但经过三年零散式的学习（如跟着老师阅读一段时间、参加相关的讲座、学术会议，以及阅读网络上的文章），以及我这段时间的集中阅读，甚至是基于相关内容的背诵，我才感觉第一次摸到了门槛，慢慢能感到和这些所谓的理论家有所共鸣（也可能是某些理论家写得比较好，比如齐格蒙特·鲍曼对现代人生活的不确定性、恐惧、消费、爱情的讨论），虽然我不清楚，究竟是我被他们卷入了一套话语体系，还是他们确实诚恳地提出他们的看法，而我终于能理解。但总的来说，这些都给我打开了一扇不同于过去思考生活的大门。

送你我一段话：

> “一本书读完可能很快就忘记了，好比竹篮打水，是一场空。但是竹篮经过一次次水的洗礼，会一次比一次干净。”

**Reply by XiangyunHuang** [reply]
可能不能完全归于这本书对我的影响。一方面书中好些地方（蛤蟆的整个经历）确实刺到我了，另一方面，我又正好一段时间来受困于与父母的关系处理上。书的内容写得实在，我第一次接触心理学，有新鲜感，又感觉到实用。

**Reply by XiangyunHuang** [reply]
> 我到现在也还没遇到过能对自己产生极大影响的书籍。

2024 年到目前为止，对我影响非常大的另一本书是陈忠实的《白鹿原》，让我对中共党史、国史和中国乡村很有兴趣，找了几本相关书，还没看完，陆陆续续写了点，但是读后感还没写完。

> 另，我想，阅读社会学和人类学这一类的书籍可能需要一个适应的过程。

我觉得我可能并不是对社会学理论感兴趣，只是寄希望于它可以解释社会上光怪陆离的社会现象，至少也提供一个社会学的分析视角，表达的方式最好可以深入浅出。我之前听完一个课程，北京大学卢晖临[《社会学概论》](https://www.bilibili.com/video/BV1XK4y1n7gp/) ，属于概论或者说导论性质的，我比较受用。

我有点好奇，有没有国外理论和中国当今现实相结合的社会学著作？在费孝通、梁漱溟、晏阳初之后，结合中国特色的新的东西，你有推荐的吗？

**Reply by XiangyunHuang** [reply]
我以前看过流传非常广的周孝正的一个视频，我也好奇，研究社会学是不是非常危险，禁忌很多？

**Reply by residualsun1** [reply]
> 我有点好奇，有没有国外理论和中国当今现实相结合的社会学著作？在费孝通、梁漱溟、晏阳初之后，结合中国特色的新的东西，你有推荐的吗？

梁漱溟和晏阳初的作品我都没看过，有些惭愧，后续要补补课，因此还没办法根据他们的风格给你做推荐。不过，国外理论和中国现实结合的著作应当是有很多的，但我的阅读量还很匮乏，只能试着从我已有的知识储备尝试回答。

”中国当今现实“是一个太大的命题，我还是一个被象牙塔保护的大学生，严格来说对真正的社会现实是不了解的，只有个体的朦胧感受，根据这份感受，我以国外理论家提出的议题作为思考的脚手架来寻找那些我过去没注意的、中国面临的现实：例如“流动性”、“风险”和“消费”等。关于这方面，我推荐鲍曼和哈夫纳的对话录《[将熟悉变为陌生](https://book.douban.com/subject/36409321/)》，这本册子涵盖了今天的爱情、移民、技术、宗教、气候、消费和贫富差距等等议题（我印象很深刻的是鲍曼说**神是人的不足**，人不能没有宗教），鲍曼活了九十多岁，2017年才去世，但一直在保持思考和创作，我最近正在读这本书，直观地感觉他说的也是我们正在经历的，有着很强的共鸣，看了一半我又去翻了他别的书，有《流动的爱》、《流动的恐惧》、《流动的现代性》等等，总的来说，他对现代社会的特征可以概括为“流动”——一切都不再稳定，但在变化之后，却不再像过去一样有一个能给大家提供方向的力量（过去是国家或市场），不仅是国家，连市场都失去了有效的力量——这不仅是西方已经在面对的问题，我感觉也和当下中国就业前景黯淡（主要从我的角度看）、大家都想进体制，然而看起来无论是延迟退休、扩大招生，都不过是饮鸩止渴很契合。这本小书也没有黑话，我觉得可以入门，这段时间它给了我对生活一些不同的理解。

如果要再更贴近中国语境的话，我推荐吴琦和项飙的对谈《[把自己作为方法](https://book.douban.com/subject/35092383/)》，在我眼中，项飙有些特殊，他在国内是极其出圈的人类学者，这很大原因是在于他关注年轻人的境况，而且我能在他的文字中感受到一种很朴素的感觉，如同我看费孝通的《乡土中国》那样。项飙很擅长讲述，一个平平无奇的概念，他可以讲得非常丰富，结合的还是我们日常生活中很熟悉、亲切的例子，例如他前段时间提的“附近”，后来又提“承认”、“认得”等等，都扎根于我们熟悉的日常生活（如果想稍微了解，可以简单看看澎湃上面的两篇文章《[项飙专访（上）︱重建附近:年轻人如何从现实中获得力量？](https://m.thepaper.cn/newsDetail_forward_24823590)》《[项飙专访（下）︱年轻人在寻找自己在世界上的存在方式和意义](https://www.thepaper.cn/newsDetail_forward_24823595)》可以感受到项飙的叙述风格和思考方式）。《把自己作为方法》不是一本研究著作，是两个人的对谈，严格来说是项飙对自己过去的回顾，但我觉得这种回顾里有不少适合科普入门认识社会学和人类学的思考，例如关于今天年轻人为什么丧、年轻人眼中中心和边缘关系的探讨等。

以上两者及其著作，即鲍曼的《将熟悉变为陌生》，以及项飙的《把自己作为方法》，共同点应该都是符合深入浅出，而要说与中国现实符合，前者在宏观上比较契合，后者则在微观上容易引起年轻人特别是大学生共鸣。但这也只是我理解的中国现实，如果要严肃一些，我暂时只能想到“劳工研究”，比如这段时间比较出圈的：孙萍花了七年调查写成的《过渡劳动：平台经济下的外卖骑手》（提起这本，是因为我记得你之前在美团做过数据分析，所以我想，你可能会对这里的语境感到熟悉）；在很早之前，香港学者潘毅写了一本《中国女工：新兴打工者主体的形成》，研究大陆工厂里的女性劳工（我后来才听说，劳工研究在国内比较敏感），用了一些不少国外理论，她做了这份研究后觉得愧对女工们，认为自己做了研究，却对她们的处境没有任何改善。最后一本是周雪光老师的《中国国家治理的制度逻辑》（因为看到你说对中共党史、国史和乡村感兴趣，这本书或许可以读读，但会比较抽象），这本书在大陆好像已经被禁了。

> 我以前看过流传非常广的周孝正的一个视频，我也好奇，研究社会学是不是非常危险，禁忌很多？

阿，我感觉你问我这个问题非常抬举我了இ௰இ，我觉得我完全没有资质回答这个问题，没有扎实的学术训练，没有足够的阅读，更没有深耕于现实调查的经历，但是，我很愿意和你分享我自己关于这个问题的看法，但是，这仅仅只是我的个人“偏见”，并不是一个社会学学生对这个问题的看法，因为我觉得我还远远没有能力以一个社会学的学生身份回答这个问题。

我对中国现实没有清晰、全面、客观的认识，但我认为它实际上很可能是狰狞的、晦暗的、灰色的。因此，我认为好的社会学研究会有痛感，会揭开社会的伤疤，但越好的研究可能越难~被允许~做出来。当下的环境不断提倡正面、积极、“筑牢”，在统一的指挥棒挥舞下，官方意识是宰制性的。因此，在中国官方语境里，我眼中的好的社会学研究其实是危险的，而能被大家看到的、被宣传的好的社会学研究则往往是能服务于建设方向和口号，而不是直面问题。（不过并不绝对，还是会有一些很好的研究能出现的）

**Reply by XiangyunHuang** [reply]
我最近看了几篇北大卢晖临教授主页上的文章，好巧他就是主要研究中国乡村社会的，解了我一些困惑。

另外，我顺便一问，你在统计之都论坛的ID是啥？我不记得我给你解答过什么问题了。

**Reply by yihui** [reply]
- https://d.cosx.org/u/residual-sun
- https://d.cosx.org/d/424264

（我只是路过看楼上荐书的）

**Reply by residualsun1** [reply]
> 另外，我顺便一问，你在统计之都论坛的ID是啥？我不记得我给你解答过什么问题了。

益辉帮我回答啦。我第一个问的是bookdown问题，后来问的是blogdown，唯独还没问过统计。

> （我只是路过看楼上荐书的）

我打算明年做一个长期专题，定期把感兴趣的社会科学领域书籍、文章和影视等集合起来，按照周报的形式发布，欢迎前来参观^^。

**Reply by XiangyunHuang** [reply]
期待你的长期专题。

**Reply by LFG12** [reply]
> https://d.cosx.org/u/residual-sun
我终于发现了统计论坛，并立即注册了账号，原来我之前访问时，只知道阅读文章，没有点开论坛，之前听yuanfan说过可以来统计之都交流，今日一看这种氛围像是互联网早期的感觉，相互帮助交流，态度友好。
在这个评论下留言，都能感受的到大家的真诚交往，期待国政能够越来越好。

**Reply by residualsun1** [reply]
LFG 你好！！欢迎光临我的小站！时间飞快，到现在，我在统计之都注册的时间也已经有两年啦。统计之都给我印象很深刻的地方就是大家都很真诚也很接地气，懂的东西很多也很乐意解答问题，互帮互助，是很难得的论坛氛围呀！嘿嘿，欢迎加入统计之都，一起入坑数据科学！

> 在这个评论下留言，都能感受的到大家的真诚交往，期待国政能够越来越好。

感谢你的祝福！我也祝愿 FG 你越来越好，在机遇遍布的深圳收获自己独特的成长！

**Comment by aric6494** [comment]
很喜欢作者的网站字体，但私自使用了你的字体配置，请问作者是否介意。

**Reply by residualsun1** [reply]
朋友你好！欢迎来到我的小站！感谢喜欢(‾◡◝)，不会介意的！我使用的字体其实也是来自博主谢益辉的哈哈，是思源宋体。

**Reply by yihui** [reply]
朋友你好！欢迎来到他的小站！感谢喜欢(‾◡◝)，我也不会介意的！

非系统自带的中文字体一般都不适合直接作为网络字体使用，因为字体文件体积太大，而我是对思源宋体按照我自己的网站用到的汉字取了个子集：https://github.com/yihui/cron/blob/main/fonts/chars.txt 然后再通过字体工具抽取了字体文件相应的子集：https://github.com/yihui/cron/blob/main/.github/workflows/fonts.yaml 这样得到的字体文件就小多了、可以通过网络加载：https://github.com/yihui/cron/blob/main/fonts/SourceHanSerifCN-Regular-yihui.woff2

如果你用到的汉字范围比我小，你可以构建你自己的子集。

**Reply by aric6494** [reply]
> 朋友你好！欢迎来到我的小站！感谢喜欢(‾◡◝)，不会介意的！我使用的字体其实也是来自博主谢益辉的哈哈，是思源宋体。

谢谢(・∀・)

**Reply by aric6494** [reply]
> 朋友你好！欢迎来到他的小站！感谢喜欢(‾◡◝)，我也不会介意的！
> 
> 非系统自带的中文字体一般都不适合直接作为网络字体使用，因为字体文件体积太大，而我是对思源宋体按照我自己的网站用到的汉字取了个子集：https://github.com/yihui/cron/blob/main/fonts/chars.txt 然后再通过字体工具抽取了字体文件相应的子集：https://github.com/yihui/cron/blob/main/.github/workflows/fonts.yaml 这样得到的字体文件就小多了、可以通过网络加载：https://github.com/yihui/cron/blob/main/fonts/SourceHanSerifCN-Regular-yihui.woff2
> 
> 如果你用到的汉字范围比我小，你可以构建你自己的子集。

同样感谢(・∀・)

**Comment by xiaoyi2964-debug** [comment]
朋友您好！在搜索格尔茨的时候无意间看到你的博文，写的非常有启发性。然后看了你的主页（有点偷窥的意味，还请见谅），非常喜欢你的行文风格，也能感受到你的真诚。祝你学业顺利，也愿你能一直写下去！

**Reply by residualsun1** [reply]
朋友你好！欢迎来到我的小站（握手🤝），十分感谢你的肯定！我觉得自己的写作还在模仿的阶段，但你的勉励给了我很多动力！说起来有些不好意思，因为将自己的写作放到网上，就意味着要做好随时被看见的准备，所以不会有「偷窥」这样的说法！欢迎你随时来访(oﾟvﾟ)ノ

**Comment by earfanfan** [comment]
@residualsun1 我刚刚给你写了信，记得抽空去瞅瞅啊。

**Reply by yihui** [reply]
我猜这应该是国政收到的第一封读者来信吧，看完记得去买点烟花爆竹回来隆重地放一下！

**Reply by residualsun1** [reply]
@earfanfan 哇咔咔，我已经收到信并回信啦，记得摸鱼去瞅瞅嘞。

> 我猜这应该是国政收到的第一封读者来信吧，看完记得去买点烟花爆竹回来隆重地放一下！

益辉呐，我看你最近在咱们的评论区都很活跃，是不是该考虑耕博啦？家里的菜园子很重要，网上的花园也需要经营一下嘞。另，现在还没烟花爆竹卖嘞，不过我最近在学一首吉他曲子，感觉能弹下来了，就是右手节奏型不好，高音也唱不上去，等我练熟了给你们献丑，以示庆祝😎。

**Reply by yihui** [reply]
活跃主要是因为你牵动着我们千千万万读者的心呐，你看你不露面的地方我有发过这么多言咩。不过既然你已尘埃落定，我就该回自己窝里了，昨晚已经开始打扫我家里的灰尘。

说起吉他曲，《加州招待所》（Hotel California）的吉他伴奏很好听，坐等你哪天把它弹唱出来，保证把走过路过的美眉们迷得五十八迷三十六道的！另一首我喜欢的吉他伴奏是苏慧伦《秋天的海》，不过这首对你可能太简单了点，也不适合男声唱。

**Reply by earfanfan** [reply]
>等我练熟了给你们献丑，以示庆祝

哈哈，这下不管练熟没练熟，元旦都要献个丑了。

@yihui  你也逃不过滴。

**Reply by yihui** [reply]
我又想起来另一首很厉害的歌，陈涌海老师自弹自唱的《[将进酒](https://www.bilibili.com/video/BV1qb411N7rh/)》，国政可否拿下？

**Reply by residualsun1** [reply]
益辉，《加州招待所》难度还是高了，不过我听了陈涌海老师的《将进酒》后找到了谱子（虽然可能不完全一样），练了一小会儿，很生疏，但右手节奏弹起来非常痛快！这首歌应该用四个和弦就可以弹完，而且右手节奏型不是很复杂，花一些时间应该可以弹下来，难的地方是弹出一种「豪气」和「狂」的感觉——我被视频里的陈涌海老师折服了。真不得不佩服古人，也不得不佩服陈老师弹唱时的一腔豪气，虽然这样说可能有点夸张，但是接触点更豪气、宏大的东西，好像会让总是纠结儿女情长和矫情的我重新感受到人还可以有很多种情感和表达方式。

想想，如果能拿下这首曲子，喝过酒后再高歌一曲，那肯定更是痛快。不过为谨慎起见，我不敢承诺能否拿下，但我会尝试！！
